src/strategies/latency.js

```diff
@@ -173,17 +173,17 @@ class LatencyStrategy extends BaseStrategy {
 
 		if (info.nodeList.length == 0) {
 			// only remove the host if the last node disconnected
-			this.broker.localBus.emit("$node.latencySlave.removeHost", node.hostname);
+			this.broker.localBus.emit("$node.latencyReplica.removeHost", node.hostname);
 			this.hostMap.delete(node.hostname);
 		}
 	}
 
-	// Master + Slave
+	// Master + Replica
```

Comment:
if you change "Slave" for "Replica" you could also change "Master"

Reply:
https://www.vice.com/en_us/article/8x7akv/masterslave-terminology-was-removed-from-python-programming-language

Reply:
I agree (If it isn't a much bigger breaking change)